lib/logstash/filters/translate.rb

```diff
@@ -81,7 +81,12 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
 
   # When using a dictionary file, this setting will indicate how frequently
   # (in seconds) logstash will check the dictionary file for updates.
-  config :refresh_interval, :validate => :number, :default => 300
+  config :refresh_interval, :validate => :number, :default => 5
+  # when using a dictionary file, if this is set to true, the dictionary's
+  # modification time will be checked before reloaded.  If the dictionary
+  # isn't modified, the refresh won't happen.  If you set this to false,
+  # increase the refresh_interval.
+  config :refresh_only_if_modified, :validate => :boolean, :default => true
```

Comment:
the only case where i think you'd want to revert back to the old behavior is on Windows.  There is a case where windows won't update the mtime of a file until it's closed... https://blogs.technet.microsoft.com/asiasupp/2010/12/14/file-date-modified-property-are-not-updating-while-modifying-a-file-without-closing-it/